e2e/scenario/base_vhd_catalog.json

```diff
@@ -13,6 +13,10 @@
         "gen2containerd": {
             "artifactName": "2204-gen2-containerd",
             "resourceId": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1699020621.21398"
+        },
+        "gen2containerdprivatekubepkg": {
```

Comment:
we shouldn't need a new VHD here, would just reuse ubuntu 2204 and supply the custom package URLs

Reply:
Using the pre-built image here to test provisioning with VHDs with private packages.